gbdx_auth/gbdx_auth.py

```diff
@@ -24,26 +54,19 @@ def session_from_envvars(auth_url='https://geobigdata.io/auth/v1/oauth/token/',
                       pull the configuration from.  Change the
                       template values if your envvars differ from the
                       default, but make sure the keys remain the same.
-
-    NOTE: There are two ways to create a GBDX session from environment variables.
-    Set an env variable GBDX_ACCESS_TOKEN to be a GBDX oauth access_token (jwt).
-    Or, provide GBDX_USERNAME, GBDX_PASSWORD, GBDX_CLIENT_ID, and
-    GBDX_CLIENT_SECRET as environment variables.
     """
     def save_token(token):
         s.token = token
 
-    if os.environ.get('GBDX_ACCESS_TOKEN'):
-        s = OAuth2Session(client_id, client=LegacyApplicationClient(client_id, access_token=os.environ.get('GBDX_ACCESS_TOKEN')))
```

Comment:
I removed this `if` because it is broken code that I merged recently. `client_id` is never defined in the `if os.environ.get('GBDX_ACCESS_TOKEN'):` case so this always breaks.